### DISCUSSION TITLE
Specifying a query parameter limit for the get products by ids rest admin api call doesn't work

### DISCUSSION THREAD

**Discussion by DavidAptikadir25** [kinngh/shopify-node-express-mongodb-app#115] [Q&A]
Hi, I have tried to use rest admin api to get product data with product_ids https://shopify.dev/docs/api/admin-rest/2024-01/resources/product#get-products?ids=632910392,921728736
I was able to get the products in chunks of 50. However when I tried to increase the limit to the maximum possible - 250
I again got only 50 products.

here is how I am passing the limit query parameter and making the api call:

```
      const requestBody = {
        path: `products.json?ids=${chunk.join(',')}`,
        query: { limit: CHUNK_SIZE /* 250 */ }
      }

      let data = await client.get(requestBody);
```

after I make the request I get this many results:
0|server  | data.body.products.length: 50

Can someone help, am I doing something wrong?

**Comment by kinngh** [comment]
Issues / Discussions here are for the boilerplate only. To get help for Shopify APIs and other related stuff, please use Shopify Community Forums
